src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -1483,6 +1456,18 @@ public partial interface IFormattable
     {
         string ToString(string format, System.IFormatProvider formatProvider);
     }
+    public readonly partial struct Index : System.IEquatable<System.Index>
```

Comment:
From https://github.com/dotnet/corefx/pull/33352

cc @tarekgh , please review for correctness

Reply:
Index and Range looks good to me. thanks